crane/views/v1.py

```diff
@@ -47,6 +47,22 @@ def ping():
     response.headers['X-Docker-Registry-Standalone'] = True
     return response
 
+@section.route('/users', methods=['GET', 'POST'])
+@section.route('/users/', methods=['GET', 'POST'])
+def users():
+    """
+    Undocumented API path required for docker login when apache configured for auth
+    See get_post_users()
+    https://github.com/docker/docker-registry/blob/master/docker_registry/index.py
+    """
+    if request.method == 'GET':
+        response = current_app.make_response(json.dumps(True))
```

Comment:
The docker-registry returns the string "OK" serialized to json as the body for this response, based on the link you provided above. I doubt it matters, but we may as well do the same.